crawler/src/test/java/com/javaacademy/crawler/common/model/BookModelTest.java

```diff
@@ -61,6 +62,18 @@ public void testEquals() {
 
     public void toStringTest() {
         BookModel bookModel = new BookModel();
-        assertEquals(bookModel.toString(), "BookModel(title=null, subtitle=null, authors=null, industryIdentifiers=null, categories=null, smallThumbnail=null, canonicalVolumeLink=null, saleability=null, listPriceAmount=0.0, listPriceCurrencyCode=null, retailPriceAmount=0.0, retailPriceCurrencyCode=null)");
+        assertEquals(bookModel.toString(),
+                "BookModel(industryIdentifier=null, subtitle=null, title=null, categories=null, authors=null, canonicalVolumeLink=null, saleability=null, smallThumbnail=null, listPriceAmount=0.0, retailPriceAmount=0.0, listPriceCurrencyCode=null, retailPriceCurrencyCode=null)");
+    }
+
+    @Test
+    public void testThis() {
```

Comment:
testThis? What does it test? I don't know.

Also, you don't have to add "Test" in every method name.

Reply:
+1